@utahdts/utah-design-system-header/src/@types/jsDocTypes.d.js

```diff
@@ -68,6 +68,7 @@
  *  @property {MenuItem[]} [actionMenu] - children menus
  *  @property {ChildrenMenuTypes} [childrenMenuType] - default is "flyout"
  *  @property {string} [className] - can be used for `selected` or any other purpose
+ *  @property {string} [hideOverview] - hide the "overview" page from the main menu (top of the list link)
```

Comment:
boolean fields usually start with `is`. So maybe this property would be better named `isOverviewHidden`? `hideOverview` sounds like a function call?